File: BotsSDK/korebot/src/main/java/kore/botssdk/audiocodes/webrtcclient/General/LogI.java
package kore.botssdk.audiocodes.webrtcclient.General;

import com.audiocodes.mv.webrtcsdk.log.ILog;

public class LogI implements ILog
{
    @Override
    public int v(String s, String s1) {
        return Log.v(s, s1);
    }

    @Override
    public int v(String s, String s1, Throwable throwable) {
        return Log.v(s, s1);
    }

    @Override
    public int d(String s, String s1) {
        return Log.d(s, s1);
    }

    @Override
    public int d(String s, String s1, Throwable throwable) {
        return Log.d(s, s1);
    }

    @Override
    public int i(String s, String s1) {
        return Log.i(s, s1);
    }

    @Override
    public int i(String s, String s1, Throwable throwable) {
        return Log.i(s, s1);
    }

    @Override
    public int w(String s, String s1) {
        return Log.w(s, s1);
    }

    @Override
    public int w(String s, String s1, Throwable throwable) {
        return Log.w(s, s1);
    }

    @Override
    public int e(String s, String s1) {
        return Log.e(s, s1);
    }

    @Override
    public int e(String s, String s1, Throwable throwable) {
        return Log.e(s, s1);
    }
}

File: BotSdk/ui/src/main/java/com/kore/ui/audiocodes/webrtcclient/general/LogI.kt
package com.kore.ui.audiocodes.webrtcclient.general

import com.audiocodes.mv.webrtcsdk.log.ILog

class LogI : ILog {
    override fun v(s: String, s1: String): Int {
        return Log.v(s, s1)
    }

    override fun v(s: String, s1: String, throwable: Throwable): Int {
        return Log.v(s, s1)
    }

    override fun d(s: String, s1: String): Int {
        return Log.d(s, s1)
    }

    override fun d(s: String, s1: String, throwable: Throwable): Int {
        return Log.d(s, s1)
    }

    override fun i(s: String, s1: String): Int {
        return Log.i(s, s1)
    }

    override fun i(s: String, s1: String, throwable: Throwable): Int {
        return Log.i(s, s1)
    }

    override fun w(s: String, s1: String): Int {
        return Log.w(s, s1)
    }

    override fun w(s: String, s1: String, throwable: Throwable): Int {
        return Log.w(s, s1)
    }

    override fun e(s: String, s1: String): Int {
        return Log.e(s, s1)
    }

    override fun e(s: String, s1: String, throwable: Throwable): Int {
        return Log.e(s, s1)
    }
}
